generator/app/template/package.json

```diff
@@ -3,25 +3,31 @@
   "version": "0.0.1",
   "description": "app1 microservice",
   "scripts": {
-    "start": "node ./dist/app.js",
-    "test": "mocha ./dist/test",
+    "build": "babel src -d dist --source-maps",
```

Comment:
Hmm, I removed babel-cli since it's in the base image. All these scripts at this point (except for the lib template) are just meant to work from docker.